### DISCUSSION TITLE
Can't start oap-server in idea

### DISCUSSION THREAD

**Discussion by andesmountain** [apache/skywalking#9032] [Q&A] [answered]
Recently i download the 9.0.0 source from official website, and i intended to start oap-server in idea,but there are many conflicts which seems like missing java file in project. Did i download the false version?

<img width="1428" alt="image" src="https://user-images.githubusercontent.com/25276587/167757533-51ac88d0-5eeb-44f2-87f6-5085d2f82831.png">

**Comment by Superskyyy** [comment, marked as answer]
You are missing the proto file which should be initialized when you setup the project in your IDE. Please strictly follow the building documentation below, you probably missed the submodule part/ due to network connection problem it did not initialize. https://skywalking.apache.org/docs/main/latest/en/guides/how-to-build/

**Reply by andesmountain** [reply]
tks

**Comment by Superskyyy** [comment]
And please do not use the `ideas` discussions type unless you are proposing something new.
